src/filesystem/impls/filer/FilerFileSystem.js

```diff
@@ -193,11 +193,15 @@ define(function (require, exports, module) {
                     BlobUtils.rename(oldPath, newPath);
                     BrambleEvents.triggerFileRenamed(oldPath, newPath);
                 }
-
-                callback();
+                FileSystemCache.refresh(function(){
```

Comment:
Also, your callback function needs to accept an `err` object, so that you know if it worked or failed:

```js
FileSystemCache.refresh(function(err){
```